clickhouse_sqlalchemy/drivers/native/base.py

```diff
@@ -46,7 +46,10 @@ class ClickHouseDialect_native(ClickHouseDialect):
 
     @classmethod
     def dbapi(cls):
-        return connector
+        return import_module(
```

Comment:
Why do we rewrite imports with `import_module`

Reply:
i will fix it (i thought it was more appropriate way to use imported modules in methods)